### DISCUSSION TITLE
where did the make file go?

### DISCUSSION THREAD

**Discussion by noob1993** [pystardust/ani-cli#496] [General] [answered]
does anybody know?

pulled changes this evening and went to build the new executable, and got a make error, can someone explain whats going on

**Comment by RaynardGerraldo** [comment, marked as answer]
> does anybody know?
> 
> pulled changes this evening and went to build the new executable, and got a make error, can someone explain whats going on

We removed the Makefile since it was causing more trouble than it's worth, refer to #417.
You can now install ani-cli on your platform with the installation docs in README

**Reply by noob1993** [reply]
okay thank you. Been kinda enjoying the cat and mouse game played, its been a great learning experience.
